iofog-agent-daemon/src/main/java/org/eclipse/iofog/field_agent/FieldAgent.java

```diff
@@ -1077,6 +1085,12 @@ private void getFogConfig() {
                 if (!Configuration.getLogLevel().equals(logLevel))
                     instanceConfig.put(LOG_LEVEL.getCommandName(), logLevel);
 
+                if (Configuration.getDockerPruningFrequency() != dockerPruningFrequency)
```

Comment:
Set it to default/keep existing value if the provided frequency is less than 1

Reply:
updated to keep existing value only